server/src/main/java/org/cloudfoundry/identity/uaa/oauth/KeyInfoBuilder.java

```diff
@@ -6,9 +6,9 @@
 public class KeyInfoBuilder {
 
     public static KeyInfo build(String keyId, String signingKey, String uaaUrl) {
-        return build(keyId, signingKey, uaaUrl, null);
+        return build(keyId, signingKey, uaaUrl, null, null);
```

Comment:
I would prefer to have a method override that would prevent us from passing 2 `nulls` in here, it makes the code hard to read.

Reply:
done